tests/de/gurkenlabs/litiengine/util/io/AlphanumComparatorTests.java

```diff
@@ -0,0 +1,67 @@
+package de.gurkenlabs.litiengine.util;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+
+import java.util.Comparator;
+
+/**
+ * This is an updated version with enhancements made by Daniel Migowski,
+ * Andre Bogus, and David Koelle. Updated by David Koelle in 2017.
+ *
+ * To use this class:
+ * Use the static "sort" method from the java.util.Collections class:
+ * Collections.sort(your list, new AlphanumComparator());
+ */
```

Comment:
I don't think this comment is intended to be here :)

Reply:
Very true! Removed